build.proj

```diff
@@ -3,8 +3,11 @@
   <PropertyGroup>
     <!-- Capture OSGroup passed to command line for setting default FilterToOSGroup value below -->
     <_OriginalOSGroup>$(OSGroup)</_OriginalOSGroup>
-    <TestOsTarget Condition="'$(OSGroup)'!=''">$(OSGroup)</TestOsTarget>
-    <TestOsTarget Condition="'$(OSGroup)'==''">$(OSEnvironment)</TestOsTarget>
+  </PropertyGroup>
+  <PropertyGroup>
+    <InputOSGroup>$(OSGroup)</InputOSGroup>
+    <InputOSGroup Condition="'$(InputOSGroup)'==''">$(FilterToOSGroup)</InputOSGroup>
+    <InputOSGroup Condition="'$(InputOSGroup)'==''">$(OS)</InputOSGroup>
```

Comment:
Sorry I should have taken a closer look at where we import dir.props in this file. You were using the right one, however with your addition to default InputOSGroup to OSEnviroment in dir.props you can no just remove this fallback here.